interop-testing/src/main/java/io/grpc/testing/integration/XdsTestServer.java

```diff
@@ -130,7 +130,8 @@ private TestServiceImpl() {
       try {
         host = InetAddress.getLocalHost().getHostName();
```

Comment:
`host` can be final now.  But not a big deal.

Reply:
done